badgerdb/badgerdb.go

```diff
@@ -97,6 +98,38 @@ type Options struct {
 	// Directory for storing the DB files.
 	// Optional ("BadgerDB" by default).
 	Dir string
+	// Uses in-memory storage, ignores file-specified options.
+	// Optional (false by default).
+	InMemory bool
+	// When SyncWrites is true all writes are synced to disk.
+	// Optional (false by default).
+	SyncWrites bool
+	// When ReadOnly is true the DB will be opened on read-only mode.
+	// Multiple processes can open the same Badger DB.
+	// Optional (false by default).
+	ReadOnly bool
+	// Truncate indicates whether value log files should be truncated to delete corrupt data, if any.
+	// This option is ignored when ReadOnly is true.
+	// Optional (false by default).
+	Truncate bool
+	// This value specifies how much data cache should hold in memory. A small size of cache means lower
+	// memory consumption and lookups/iterations would take longer. Setting size to zero disables the
+	// cache altogether.
+	// Optional (1 GB by default).
+	MaxCacheSize int64
+
+	// Encryption related options.
+
+	// EncryptionKey is used to encrypt the data with AES. Type of AES is used based on the key
+	// size. For example 16 bytes will use AES-128. 24 bytes will use AES-192. 32 bytes will
+	// use AES-256.
+	// Optional (empty by default).
+	EncryptionKey []byte // encryption key
```

Comment:
Can the `// encryption key` comment be removed from this line?

But this is a great feature of BadgerDB v2 and it's great to be able to use it with `gokv`! I haven't read the v2 release notes before, so I didn't know about this feature yet, so thanks for incorporating it!